src/main.cpp

```diff
@@ -74,6 +74,13 @@ int main(int argc, char* argv[])
 
   bool ok = app.init("RALibRetro", 640, 480);
 
+  parsedArgs parsedArgs = app.parseArgs(argc, argv);
+  app.logger().info("[ARGS] core: %s", parsedArgs.core.c_str());
+  app.logger().info("[ARGS] system: %i", parsedArgs.system);
+  app.logger().info("[ARGS] game: %s", parsedArgs.game.c_str());
+
+  ok &= app.tryLoadContent(parsedArgs);
```

Comment:
suggest wrapping this new code in an
```
if (ok && argc > 1) {
```
block to avoid logging
```
[INFO] [ARGS] core: 
[INFO] [ARGS] system: 0
[INFO] [ARGS] game: 
```
Also, recommend just using "[APP]" as the category.

Reply:
VisualStudio doesn't like something about the way this is interacting with the `__try` below:
```
1>e:\source\retroachievements\ralibretro\src\main.cpp(71): error C2712: Cannot use __try in functions that require object unwinding
```
You'll probably have to move it into a function.

Reply:
@Jamiras thanks for your comments, I'll try to solve them all the next days. This one however, I can't properly test, because I don't have VisualStudio - the MinGW build runs through.

Reply:
✔️ (check if any cli args are provided, log as [APP]) https://github.com/theMK2k/RALibretro/commit/160681e1219745d515917547a2e17493dff4d350